### PR TITLE
fix sources that resulted with different column lenght in some cases

### LINKED ISSUE #2816: Charts donut NB example raise a long set of warning
See attachment:

screen shot 2015-09-05 at 14 23 26

### PR DESCRIPTION
**Pull request by fpliger**
issues: fixes #2816

### CODE REVIEW

**Comment by bryevdv** [comment]
looks like spurious failure, restarted single test.

File: bokeh/charts/builder/donut_builder.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def draw_external_ring(self, colors=None):
             start = [start_angle] + end[:-1]
             base_color = colors[i]
             #fill = [ base_color.lighten(i*0.05) for i in range(len(details) + 1) ]
```
Comment (bryevdv) [inline bokeh/charts/builder/donut_builder.py]: prefer not to check in commented code.

**Comment by bryevdv** [comment]
LGTM, will merge when tests pass.

**Comment by damianavila** [comment]
I guess this is just a shor-term fix until tne new chart iteration... am I right?
Btw, I restarted the failing build...

**Pull request merged**